examples/l3out/comprehensive/main.tf

```diff
@@ -0,0 +1,1110 @@
+              encap       = "vlan-3"
+              mac         = "00:22:BD:F8:19:FF"
+              target_dscp = "EF"
+              path_attributes = [
+                {
+                  domain_dn           = aci_physical_domain.physical_domain.id
+                  vlan                = "vlan-3"
+                  floating_address    = "10.23.2.5/12"
+                  forged_transmit     = false
+                  mac_change          = false
+                  promiscuous_mode    = false
+                  secondary_addresses = ["10.34.23.1/12", "10.34.23.2/12", "10.34.23.3/12"]
+                }
+              ]
+              bgp_peers = [
+                {
+                  ip_address          = "10.1.1.21"
+                  address_control     = ["af-mcast", "af-ucast"]
+                  allowed_self_as_cnt = "1"
+                  peer_controls       = ["bfd"]
+                  private_as_control  = ["remove-all", "remove-exclusive"]
+                  admin_state         = "disabled"
+                  route_control_profiles = [
+                    {
+                      direction = "export"
+                      target_dn = aci_route_control_profile.profile2.id
+                    }
+                  ]
+                },
+                {
+                  ip_address          = "10.1.1.42"
+                  address_control     = ["af-mcast", "af-ucast"]
+                  allowed_self_as_cnt = "1"
+                  bgp_controls = {
+                    as_override = true
+                  }
+                  peer_controls      = ["bfd"]
+                  private_as_control = ["remove-all", "remove-exclusive"]
+                  admin_state        = "enabled"
+                  route_control_profiles = [
+                    {
+                      direction = "export"
+                      target_dn = aci_route_control_profile.profile2.id
+                    },
+                    {
+                      direction = "import"
+                      target_dn = aci_route_control_profile.profile1.id
+                    }
+                  ]
+                },
+              ]
+            },
+            {
+              pod_id      = "1"
+              node_id     = "107"
+              ip_address  = "10.1.1.19/12"
+              encap       = "vlan-4"
+              mac         = "00:22:BD:F8:19:FF"
+              target_dscp = "EF"
+              path_attributes = [
+                {
+                  domain_dn           = aci_physical_domain.physical_domain.id
+                  floating_address    = "10.23.2.8/12"
+                  forged_transmit     = false
+                  mac_change          = false
+                  promiscuous_mode    = false
+                  secondary_addresses = ["10.34.23.1/12", "10.34.23.2/12", "10.34.23.3/12"]
+                }
+              ]
+            },
+          ]
+        },
+      ]
+    }
+  ]
+}
+
+# output "module"{
```

Comment:
Do we need this part?

Reply:
Not really. It's just a placeholder for the output